about-me.html

```diff
@@ -0,0 +1,172 @@
+---
+layout: default
+title: About me
+---
+
+<h1>About me</h1>
+
+<section id="about-me">
+    <p>
+        My name is Kévin and I am a software engineer specialized in backend applications.
+        I started web development as an autodidact in 2008 and acquired a strong knowledge
+        and expertise in this field over time.
+    </p>
+
+    <p>
+        I want this page to be a "<em>not-so-formal</em>" description of who I am,
+        my main skills and why I think I am a good software engineer and person to
+        work with. For a more formal introduction, you can <a href="/Resume_Kevin-Gomez-Pinto.pdf">download my résumé</a>.
+    </p>
+
+    <p>
+        <img src="/img/me_forum_php_2017.jpeg" title="Picture of myself, speaking at Forum PHP 2017" />
+    </p>
+
+    <p>
+        Software quality, scalability and Service Oriented Architecture are subjects
+        that fascinate me. I do my best to improve myself in these fields but I also
+        try to open my perspective to other domains like compilation and static analysis.
+    </p>
+
+    <p>
+        Generally speaking, I love working and sharing on topics related to software
+        architecture and software quality. I like to see myself as a passionate developer,
+        an Open Source enthusiast and an eternal learner.
+    </p>
+
+    <h2>What I am looking for</h2>
+
+    <p>
+        <b>Challenge motivates me.</b> Even if I have mainly worked on PHP and
+        Ruby applications, I crave for opportunities to discover new
+        technologies and concepts.
+    </p>
+
+    <p>
+        <b>I love learning.</b> If you use programming languages or tools that I
+        have not already mastered, I probably already heard about it and I definitely
+        am confident that I can quickly become proficient with it.
+    </p>
+
+    <p>
+        <b>I am interested in various topics.</b> Pure development, architecture and
+        clean code, DevOps practices, scalability and performance concerns, monitoring
+        and SRE are all topics that I am interested in. And there are a lot more!
```

Comment:
SRE is considered to be an "implementation of DevOps practices". Is it interesting to repeat the notion twice ?

Reply:
You're probably right, I was just name-dropping :D